modules/sildisco/src/IdPDisco.php

```diff
@@ -39,6 +41,10 @@ private function getMetadataPath()
         return __DIR__ . '/../../../metadata/';
     }
 
+    /**
+     * @throws NoState
```

Comment:
I don't see where this `NoState` error is actually being thrown. Did you intend to throw that on lines 60 and/or 65?

Reply:
PHPStorm added that. It's in `Auth\State::loadState` which is why I removed the check on `$state`.